pypeit/par/pypeitpar.py

```diff
@@ -1743,6 +1750,10 @@ def __init__(self, bspline_spacing=None, boxcar_radius=None, trace_npoly=None,
         descr['maxnumber'] = 'Maximum number of objects to extract in a science frame.  Use ' \
                              'None for no limit.'
 
+        defaults['find_trim_edge'] = [5,5]
```

Comment:
The description is incorrect. It does not trim the slit. It simply ignores objects found within this many pixels of the slit edge. Also, we should probably consider making this arc seconds.

Reply:
Look at this again, @jhennawi 

see this line in `extract.py`

```
    ximg, edgmask = pixels.ximg_and_edgemask(slit_left, slit_righ, thismask, trim_edg=trim_edg)
```

which I think has the effect of zeroing out the slit in these trimmed edges.

Reply:
It does other things with trim_edge. Look in the code;).